### PR TITLE
KYP-720: Serializer helpers tests and refactoring

### PR DESCRIPTION
**Pull request by nedick007**
Unit tests and refactoring for serialize Helper classes.

### CODE REVIEW

File: Helper/DeletedProductSerializer.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ public function serialize($deletedProductOrders)
                 $parentItemId   = $item->getParentItemId();
                 $itemId         = $item->getProductId();
                 $itemSku        = $item->getSku();
-                $itemName       = $item->getname();
```
Comment (evgeni-spasov-metrilo) [inline Helper/DeletedProductSerializer.php]: Wow. It was like that before?

Comment (nedick007) [reply Helper/DeletedProductSerializer.php]: yes

Comment (evgeni-spasov-metrilo) [reply Helper/DeletedProductSerializer.php]: Jesus... it is a bug, right?

Comment (nedick007) [reply Helper/DeletedProductSerializer.php]: Well long story short - PHP. It was a bug from semantic point of view but In PHP, Variable names are case-sensitive but function names are case-insensitive, though it is usually good form to call functions in the same case as they appear in their declaration.

Comment (evgeni-spasov-metrilo) [reply Helper/DeletedProductSerializer.php]: 🤦‍♂

**Pull request merged**